assets/js/modules/optimize/index.legacy.js

```diff
@@ -25,34 +25,11 @@ import { __ } from '@wordpress/i18n';
 /**
  * Internal dependencies
  */
-import OptimizeSetup from '../../modules/optimize/setup';
-import { fillFilterWithComponent, getModulesData } from '../../util';
+import { getModulesData } from '../../util';
 
 const slug = 'optimize';
 const modulesData = getModulesData();
 if ( modulesData.optimize.active ) {
-	/**
-	 * Add components to the settings page.
-	 */
-	addFilter(
-		`googlesitekit.ModuleSettingsDetails-${ slug }`,
-		'googlesitekit.OptimizeModuleSettingsDetails',
-		fillFilterWithComponent( OptimizeSetup, {
-			onSettingsPage: true,
-		} )
-	);
-
-	/**
-	 * Add component to the setup wizard
-	 */
-	addFilter(
-		`googlesitekit.ModuleSetup-${ slug }`,
-		'googlesitekit.OptimizeModuleSetupWizard',
-		fillFilterWithComponent( OptimizeSetup, {
-			onSettingsPage: false,
-		} )
-	);
-
 	/**
 	 * Add data to the congrats setup Win Notification for display.
 	 */
```

Comment:
@aaemnnosttv I've not made this change pending @felixarntz response

Reply:
:+1: sounds good, adding these back in.

Reply:
@jqlee85 Sorry this may not have been clear, the code that you removed _should_ remain removed. I was only responding to @aaemnnosttv's suggestion to move the remaining code below over as well - _that_ we shouldn't do because it's still legacy code.